### ISSUE TITLE
Net6 support

### ISSUE THREAD

**Issue by tomekpr** [lucaslorentz/minicover#147]
Hi everyone!

Any chance you can add support for net6? Apologies if that's already work in progress.

**Comment by ffMathy** [comment]
Is it not working with dotnet 6 currently?

What happens when you try?

**Comment by tomekpr** [comment]
Posted this on behalf of my colleague and asked for more details and he said:

'It simply has a check at the beginning whether or not it's on the supported list, it doesn't even start'

minicover instrument --workdir ./ --assemblies source/test/**/bin/**/*.dll --sources source/app/**/*.cs
It was not possible to find any compatible framework version
The framework 'Microsoft.NETCore.App', version '5.0.0' (x64) was not found.
  - The following frameworks were found:
      6.0.1 at [/usr/share/dotnet/shared/Microsoft.NETCore.App]
You can resolve the problem by installing the specified framework and/or SDK.
The specified framework can be found at:

So the context is: job running on a linux image with net6 installed only and you get the above error, thus the assumption that net6 is not supported.

**Comment by ffMathy** [comment]
Ah. This just means that Minicover itself can't run because Minicover is currently written in dotnet 5.

But once you install the dotnet 5 runtime, it should be possible to use Minicover on dotnet 6 applications too.

That being said, we should probably update Minicover to be written in dotnet 6. That would make sense. I don't have much time these days though.

**Comment by HarrySnipe** [comment]
I'm the colleague tomekpr is talking about.

I've actually bumped into another issue trying to implement it.

$ minicover opencoverreport --threshold 60 --workdir ./
Specify --help for a list of available options and commands.
Unrecognized option '--threshold'

Again the context: Is a job running on a linux image with net6 installed, sdk 5.0 was added so that minicover could be installed, we are now just having the above error.

Thank you for your speedy reply as well :)
